Sources/MapboxCoreNavigation/NavigationService.swift

```diff
@@ -308,8 +308,8 @@ public class MapboxNavigationService: NSObject, NavigationService {
         
         let routerType = routerType ?? DefaultRouter.self
         _router = routerType.init(alongRouteAtIndex: routeIndex, in: routeResponse, options: routeOptions, directions: self.directions, dataSource: self)
-        NavigationSettings.shared.distanceUnit = routeOptions.locale.usesMetric ? .kilometer : .mile
-        
+        NavigationSettings.shared.distanceUnit = routeOptions.distanceMeasurementSystem.distanceUnit
```

Comment:
It was possible before to display metric units with an American English or British English locale, but it required setting `NavigationSettings.distanceUnit` explicitly. This makes that explicit setting unnecessary.

Reply:
I will check the docs after this merged.

Reply:
https://github.com/mapbox/mapbox-navigation-ios/issues/3587